tic-tac-toe-app/GameBoard.jsx

```diff
@@ -0,0 +1,86 @@
+const calculateWinner = (board) =>{
+  const winningConditions = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ];
+  
+  for(let i=0; i < winningConditions.length; i++){
+    const [a,b,c] = winningConditions[i];
+    if(board[a] && board[a] === board[b] && board[a] === board[c]){
+      console.log(board);
+      console.log(`The winner is player ${board[a]}`);
```

Comment:
Most users/players will not play with the developer console open. Consider removing the console.log statements prior to deploying this app to a production environment.